apps/accounts/tests/test_password_reset_while_authenticated.py

```diff
@@ -193,26 +193,15 @@ def test_password_change_reuse_validation(self):
         self.user = User.objects.get(username="fred")  # get user again so that you can see password changed
         self.assertEquals(self.user.check_password("IchangedTHEpassword#123"), True)
 
-        # add 12 minutes to time to expire current password
-        StubDate.now = classmethod(
-            lambda cls, timezone: datetime.now().replace(tzinfo=pytz.UTC) + relativedelta(minutes=+12)
-        )
-        self.client.logout()
-        form_data = {'username': 'fred',
-                     'password': 'IchangedTHEpassword#123'}
-        response = self.client.post(reverse('login'), form_data, follow=True)
-        self.assertContains(response,
-                            ("Your password has expired, change password strongly recommended."))
-
     @override_switch('login', active=True)
     @mock.patch("apps.accounts.validators.datetime", StubDate)
-    def test_password_expire_not_affect_staff(self):
+    def test_no_password_expire(self):
```

Comment:
The old test condition is no longer valid, so I reused its corpse for testing that password expiration dates are no longer checked.